docs/team/zognin.md

```diff
@@ -3,34 +3,60 @@ layout: page
 
-* **New Feature**:
+* **New Feature**: Added the ability to find items that are an exact match or lie within a range. (Pull request [#85](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/85)).
+  * What it does: Allows the user to find relevant data with a command.
+  * Justification: Users will store a lot of data after using the app for some time.
+    The find feature allows them to filter for data they need.
+  * Highlights: Implemented two generic classes to make the find feature more flexible.
+    This allows different objects to make use of the generic classes if they intend to achieve the same search behaviour,
+    which reduces code duplication.
 
-* **Code contributed**: [RepoSense link]()
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=zognin&tabRepo=AY2122S1-CS2103T-W16-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
 
-* **Project management**:
+* **Team tasks**:
+  * Made the release of `v1.2` and `v1.3` on GitHub.
+  * Set up the codebase so that the team can implement new features in a consistent manner and in parallel.
+    (Pull requests [#34](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/34), [#56](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/56)).
+  * Maintain the issue tracker by creating issues, adding labels and closing issues.
 
 * **Enhancements to existing features**:
+  * Implemented the overall GUI such as styling and tabs. (Pull request [#66](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/66)).
+
+* **Review/mentoring contributions**:
+  * PRs reviewed: [#69](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/69),
+    [#84](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/84),
+    [#120](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/120),
+    [#159](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/159).
+  * Planned tasks and facilitated task distribution.
 
 * **Documentation**:
-    * User Guide:
-    * Developer Guide:
+  * User Guide:
+    * Added some recipe features. (Pull Request [#156](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/156)).
+    * Updated feature overview. (Pull Request [#233](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/233)).
+  * Developer Guide:
+    * Added user stories. (Pull Request [#78](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/78)).
+    * Added implementation details for the switching of tabs and editing of ingredients in orders/ recipes. (Pull Request [#158](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/158)).
+    * Added some manual testing instructions. (Pull Request [#239](https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/239)).
 
 * **Community**:
```

Comment:
Do you think bugs found during ped can be counted here?

something like this?
<img width="910" alt="Screenshot 2021-11-02 at 2 29 07 PM" src="https://user-images.githubusercontent.com/77243938/139797020-f7efb8d1-ca90-4fb6-97cc-618439f5e72d.png">

Reply:
I think can, but I don't want to include it for my PPP